src/res/translation/translation_de_DE.ts

```diff
@@ -1299,42 +1300,42 @@
-        <translation type="unfinished"></translation>
+        <translation>Einige Audiotreiber verhindern das Umstellen der Puffer Verzögerung aus einer Anwendung heraus. In diesem Falle wird die Einstellung der Puffer-Verzögerung in Jamulus deaktiviert und sollte daher im Treiber des Audiointerface selbst geändert werden. Unter Windows kann man diese Einstellung über den Knopf &quot;ASIO-Einstellungen&quot; erreichen. Unter Linux sollte man das Jack-Configuration-Tool verwenden um die Puffergrösse zu ändern.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="169"/>
         <source>If the buffer delay settings are disabled, it is prohibited by the audio driver to modify this setting from within the software. On Windows, press the ASIO Device Settings button to open the driver settings panel. On Linux, use the Jack configuration tool to change the buffer size.</source>
-        <translation type="unfinished"></translation>
+        <translation>Wenn die Puffer-Verzögerungs-Einstellungen deaktiviert sind , verbietet der Audio-Treiber die entsprechenden Änderungen aus der Anwendung heraus. In Windows betätige hierzu den Button &quot;ASIO-Einstellung&quot;. Unter Linux benutze das &quot;Jack Configuration Tool&quot; um die Puffergrösse zu ändern.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="177"/>
         <source>Sound card driver settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Soundkarten Einstellungen</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="178"/>
         <source>This opens the driver settings of your sound card. Some drivers allow you to change buffer settings, others like ASIO4ALL let you choose input or outputs of your device(s). More information can be found on jamulus.io.</source>
-        <translation type="unfinished"></translation>
+        <translation>Diese öffnet die Treiber Einstellungen der Soundkarte. Einige Treiber erlauben die Änderung der Puffer Einstellungen, andere wie z.B. ASIO4ALL ermöglichen es Dir , die Audio Ein- und Ausgänge Deiner angeschlossenen Audiogeräte auszuwählen. Weitere Informationen hierzu findest Du auf jamulus.io</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="183"/>
         <source>Opens the driver settings. Note: </source>
-        <translation type="unfinished"></translation>
+        <translation>Öffnet die Treibereinstellungen. Beachte:</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="184"/>
         <source> currently only supports devices supporting a sample rate of </source>
-        <translation type="unfinished"></translation>
+        <translation>Aktuell werden nur Geräte unterstützt die eine Sample Rate von </translation>
```

Comment:
the name Jamulus will be added before that. So I think it should be something like " unterstützt aktuell nur Geräte, die eine Sample Rate von"